`make go.test.coverage` met panic: permission denied in `darwin/arm64`

**If you are reporting *any* crash or *any* potential security issue, *do not*
open an issue in this repo. Please report the issue via [ASRC](https://security.alibaba.com/)(Alibaba Security Response Center) where the issue will be triaged appropriately.**

```
--- FAIL: TestStartWithNoError (0.00s)
panic: permission denied [recovered]
        panic: permission denied

goroutine 83 [running]:
testing.tRunner.func1.2({0x106ba7ea0, 0x108ee5388})
        /usr/local/go/src/testing/testing.go:1545 +0x274
testing.tRunner.func1()
        /usr/local/go/src/testing/testing.go:1548 +0x448
panic({0x106ba7ea0?, 0x108ee5388?})
        /usr/local/go/src/runtime/panic.go:920 +0x26c
github.com/agiledragon/gomonkey/v2.modifyBinary(0x1057e6370, {0xc0000788a0, 0x18, 0x1069235e0?})
        /Users/bitliu/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.9.0/modify_binary_darwin.go:9 +0xf4
github.com/agiledragon/gomonkey/v2.replace(0x1057e6370, 0x1069235e0?)
        /Users/bitliu/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.9.0/patch.go:269 +0xbc
github.com/agiledragon/gomonkey/v2.(*Patches).ApplyCore(0xc00060f9a0, {0x1069235e0?, 0x107059018?, 0x0?}, {0x1069235e0, 0x107059028, 0x13})
        /Users/bitliu/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.9.0/patch.go:213 +0xfc
github.com/agiledragon/gomonkey/v2.(*Patches).ApplyFunc(0x0?, {0x1069235e0, 0x107059018}, {0x1069235e0?, 0x107059028})
        /Users/bitliu/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.9.0/patch.go:83 +0x1a8
github.com/agiledragon/gomonkey/v2.ApplyFunc(...)
        /Users/bitliu/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.9.0/patch.go:25
github.com/alibaba/higress/pkg/bootstrap.TestStartWithNoError(0xc0007f8820)
        /Users/bitliu/github/higress/pkg/bootstrap/server_test.go:39 +0x110
testing.tRunner(0xc0007f8820, 0x107058f50)
        /usr/local/go/src/testing/testing.go:1595 +0x198
created by testing.(*T).Run in goroutine 1
        /usr/local/go/src/testing/testing.go:1648 +0x5dc
```

--- a/go.mod
+++ b/go.mod
@@ -15,7 +15,7 @@ replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.6
-	github.com/agiledragon/gomonkey/v2 v2.9.0
+	github.com/agiledragon/gomonkey/v2 v2.11.0
 	github.com/avast/retry-go/v4 v4.3.4
 	github.com/compose-spec/compose-go v1.8.2
 	github.com/docker/cli v20.10.20+incompatible

